### ISSUE TITLE
I'm interested in adding some features

### ISSUE THREAD

**Issue by kerkeslager** [openpokertools/openpokertools.com#1]
I'm interested in adding a few features to this, mainly so that I can use them myself! But I wanted to discuss the features first before I work on them because I don't want to put a lot of work into something that doesn't fit your vision and have you not accept the PR.

The first feature I'd like to add is adding the ability to input suits to the equity calculator ranges. This doesn't matter preflop because for example J♠T♠ and J♡T♡ play are identical, but it matters a great deal for the flop and later. For example on an 2♣9♡8♡ flop, J♡T♡ should be in range to call most shoves, whereas J♠T♠ is a mediocre straight draw that shouldn't be in your range to call most overbets. Just having "JTs" as your range doesn't allow you to simulate this situation as part of a range.

My thinking is to:

1. Trim down the current display to use radio boxes for each player to select between "cards" and "range", and only display what's selected with the radio button.
2. Add a third radio button called "Advanced range".
3. When the "Advanced range" radio button is selected, display an advanced range selector which allows you to select combos.
4. I'm thinking the combo selection looks like a long list of combos sorted by high card first then low card, with tables for each combination of ranks. Pairs would be a row with 6 checkboxes looking like this:

A♠A♡ | A♠A♢ | A♠A♣ |  A♡A♢ | A♡A♣ | A♢A♣ 

All non-pairs would look like this, with checkboxes to select an entire column or row, and a checkbox on the corner to select the diagonal of all suiteds:

A♠K♠ | A♠K♡ | A♠K♢ | A♠K♣
A♡K♠ | A♡K♡ | A♡K♢ | A♡K♣
A♢K♠ | A♢K♡ | A♢K♢ | A♢K♣
A♣K♠ | A♣K♡ | A♣K♢ | A♣K♣

Do you agree with this design? Any suggestions where I should start looking in the code to implement this?

**Comment by kerkeslager** [comment]
@openpokertools Yeah, that's what I'm thinking of being the "card selector". I'll have you an HTML file (I'm generating a lot of it with Python just because I'm better with that than JS).

**Comment by kerkeslager** [comment]
@openpokertools How do you feel about the idea of a right-click (two finger click on Mac)? I played around with a few ideas and didn't like most of them. The two ideas that seemed best to me were:

1. Add a right click menu to the existing range tool. When you right click a square, such as AKo, it would bring up a menu with all the combos of AKo, allowing you to select the ones you want with checkboxes. Boxes which have been modified in this way would have some "indeterminate" display which indicates to the user that part of that bucket has been selected rather than the whole thing being selected or unselected.
2. The only other idea I could come up with seems like it's probably easier to use, but more tedious: a whole new menu which allows you to select two cards and then add the resulting combo into the range. This would be extremely tedious for large ranges (such as all Ahxh).

**Comment by openpokertools** [comment]
right click sounds reasonable to me. The second question is how to indicate that the combos for a selection have been modified?

**Comment by openpokertools** [comment]
Thinking about it more, right click is a GREAT idea. 🥇

**Comment by kerkeslager** [comment]
@openpokertools Re: how to indicate that combos for a selection have been modified: I think we could add a bit of space below the letters. For suited stuff like ATs, we can just show the suits. For everything else, we can show up to two combos with suits overlapping, i.e. for AKo, a heart with a club peeking out from behind it on the right side indicates AhKc. If 3 or more combos are included, we can show the first one, with (for example) +3 to show that there are 3 more combos we don't have room to display.

Does that make sense?

**Comment by openpokertools** [comment]
Ya I like it. I think you can go ahead implementing the rendering. We'll work on the logic after.

**Comment by kerkeslager** [comment]
I'll have some time to work on it this week.

**Comment by openpokertools** [comment]
This is now implemented. Thanks for the suggestion.
